docs/development/getting-started-locally.md

```diff
@@ -0,0 +1,162 @@
+
+    ```yaml
+    # Configuration for storage provider
+    store:
+        secretRef:
+            name: etcd-backup-secret-name
+        container: object-storage-container-name
+        provider: aws # options: aws,azure,gcp,openstack,alicloud,dell,openshift,local
+        prefix: etcd-test
+    ```
+
+    Brief explanation of keys:
+
+    - `secretRef.name` is the name of the secret that was applied as mentioned above
+    - `store.container` is the object storage bucket name
+    - `store.provider`  is the bucket provider. Pick from the options mentioned in comment
+    - `store.prefix`    is the folder name that you want to use for your snapshots inside the bucket.
+
+### Applying the Etcd CR
+
+> **Note:** With backups enabled, make sure the bucket is created in corresponding cloud provider before applying the Etcd yaml
+
+Create the Etcd CR (Custom Resource) by applying the Etcd yaml to the cluster
+
+```sh
+# Apply the prepared etcd CR yaml
+kubectl apply -f config/samples/druid_v1alpha1_etcd.yaml
+```
+
+### Annotate Etcd CR with the reconcile annotation
+
+> **Note :** If the `ignore-operation-annotation` flag is set to `true`, this step is not required
+
+The above step creates an Etcd resource, however etcd-druid won't pick it up for reconciliation without an annotation. To get etcd-druid to reconcile the etcd CR, annotate it with the following `gardener.cloud/operation:reconcile`.
+
+```sh
+# Annotate etcd-test CR to reconcile
+kubectl annotate etcd etcd-test gardener.cloud/operation="reconcile"
+```
+
+This starts creating the etcd cluster
+
+### Verify the Etcd cluster
+
+To obtain information regarding the newly instantiated etcd cluster, perform the following step, which gives details such as the cluster size, readiness status of its members, and various other attributes.
+
+```sh
+kubectl get etcd -o=wide
+```
+
+#### Verify Etcd Member Pods
+
+To check the etcd member pods, do the following and look out for pods starting with the name `etcd-`
+
+```sh
+kubectl get pods
+```
+
+#### Verify Etcd Pods' Functionality
+
+Verify the working conditions of the etcd pods by putting data through a etcd container and access the db from same/another container depending on single/multi node etcd cluster
+
+```sh
+# Put a key-value pair into the etcd 
+kubectl exec -it <etcd_pod> -c etcd -- etcdctl put key1 value1
+# Retrieve all key-value pairs in the etcd db
+kubectl exec -it <etcd_pod> -c etcd -- etcdctl get --prefix ""
+```
+
+If multinode etcd cluster, insert the key-value pair from `etcd` container of one etcd member and retrieve from `etcd` container of another to verify consensus between the multiple etcd members.
+
+#### View Etcd Database File
+
+To inspect the Etcd database file, execute the following command in the `backup-restore` container of any Etcd pods to check the database file located at `var/etcd/data/new.etcd/snap/db`
+
+```sh
+kubectl exec -it <etcd_pod> -c backup-restore -- cat var/etcd/data/new.etcd/snap/db
```

Comment:
Same as above. You need not specify the steps to get into the container, but simply link to the previous section for that information.